react/OrderItems.tsx

```diff
@@ -65,11 +65,12 @@ const updateTotalizersAndValue = ({
   const oldItemSellingPrice = oldItem.sellingPrice ?? 0
 
   const oldPrice = oldItemPrice * oldItemQuantity
-  const newPrice = newItem.price * newItem.quantity
+  const newPrice = newItem.price! * newItem.quantity
   const subtotalDifference = newPrice - oldPrice
 
   const oldDiscount = (oldItemSellingPrice - oldItemPrice) * oldItemQuantity
-  const newDiscount = (newItem.sellingPrice - newItem.price) * newItem.quantity
+  const newDiscount =
+    (newItem.sellingPrice! - newItem.price!) * newItem.quantity
```

Comment:
Same thing here.
I could be really obvious 😂 But maybe that's worth a comment?

Reply:
same as above :smile: